src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -56,11 +56,26 @@ public function getMaxDepth();
      */
     public function setSerializedName(string $serializedName = null);
 
+    /**
+     * Adds the serialization name for this attribute.
+     */
+    public function addSerializedName(string $serializedName, array $groups = []);
```

Comment:
IMHO should this one should be `setSerializedNames` (no need for an adder) - which simplifies resetting

setSerializedName(null) previous, vs setSerializedNames([]) now